src/libraries/Microsoft.PowerFx.Core/Lexer/BinaryOp.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Microsoft.PowerFx.Core.Lexer
 {
-    internal enum BinaryOp
+    public enum BinaryOp
```

Comment:
>public [](http://example.com/codeflow?start=4&length=6)

nit - can you add the /// comments for these new public types (it's probably giving a warning for them)